ui/src/app/views/dataset-page.tsx

```diff
@@ -162,12 +159,14 @@ export const ValueListItem: React.FunctionComponent <
     </div>;
   };
 
-const plusLink = (dataTestId: string, path: string) => {
-  return <a data-test-id={dataTestId} href={path}
+const plusLink = (dataTestId: string, path: string, disable: boolean) => {
+  return <TooltipTrigger data-test-id='plus-icon-tooltip' disabled={!disable}
+                         content='Requires Owner or Writer permission'>
+    <Clickable disabled={disable} data-test-id={dataTestId} href={path}
             onClick={e => {navigateAndPreventDefaultIfNoKeysPressed(e, path); }}>
     <ClrIcon shape='plus-circle' class='is-solid' size={16}
-             style={styles.selectBoxHeaderIconLinks}/>
-  </a>;
+             style={{fill: disable ? colorWithWhiteness(colors.dark, 0.4) : colors.accent}}/>
```

Comment:
Opt: You could make the style object for this class be a function that takes disabled and returns the right color? (Would avoid getting rid of the object.)
I'm good either way on this, but I know we have a preference for JS objects.

Reply:
So since the other style in wrapped under reactStyle i had to create a new const